cips/cip-30.md

```diff
@@ -0,0 +1,108 @@
+| cip           | 30                                                                                         |
+|---------------|--------------------------------------------------------------------------------------------|
+| title         | Prevent Auto-Claiming of Staking Rewards                                                   |
+| description   | This CIP proposes removing the automatic claiming of staking rewards when modifying delegations. Instead, rewards are to be stored until manually claimed. |
+| author        | Dean Eigenmann ([@decanus](https://github.com/decanus)), Marko Baricevic ([@tac0turtle](https://github.com/tac0turtle)) |
+| discussions-to |                                 |
+| status        | Draft                                                                                      |
+| type          | Standards Track                                                                            |
+| category      | Core                                                                                       |
+| created       | 2025-01-27                                                                                 |
```

Comment:
extremely optional nit, but this is a week old

Reply:
amended